stac_fastapi/api/setup.py

```diff
@@ -1,16 +1,20 @@
 """stac_fastapi: api module."""
 
+from distutils.util import convert_path
+
 from setuptools import find_namespace_packages, setup
 
+main_ns = {}
+ver_path = convert_path("stac_fastapi/api/version.py")
+with open(ver_path) as ver_file:
+    exec(ver_file.read(), main_ns)
+
```

Comment:
Can we avoid doing this? 

I think it's fine to manually hard code the version here. 

We can then use something like `bump2version` to update all the files we want when doing a release

e.g. https://github.com/developmentseed/titiler/blob/main/.bumpversion.cfg

Reply:
ok, i will hard code it for now.
I created a follow up ticket
https://github.com/stac-utils/stac-fastapi/issues/648